crates/containerd-shim-wasm-test/build.rs

```diff
@@ -0,0 +1,108 @@
+use std::io::Write;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+use anyhow::{bail, Context, Result};
+use lazy_static::lazy_static;
+
+fn env_path(key: impl AsRef<str>) -> Result<PathBuf> {
+    std::env::var_os(key.as_ref())
+        .map(Into::into)
+        .with_context(|| format!("failed to read env-var {}", key.as_ref()))
+}
+
+lazy_static! {
+    static ref OUT_DIR: PathBuf = env_path("OUT_DIR").unwrap();
```

Comment:
What is the OUT_DIR directory? Is it `/target`?

Reply:
it's an env-var with a folder where buildscripts can store artifacts.
it's also accesible from the crate itself.